docs/admin/authentication.md

```diff
@@ -33,7 +33,9 @@ plane, must authenticate when making requests to the the API server.
 ## Authentication strategies
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
-authenticate API requests through authentication plugins. As HTTP request are
+authenticate API requests through authentication plugins. A bearer token
+must be something that can be included in an HTTP header value without any
```

Comment:
not sure this is very illuminating. would an example be better (e.g. to use the bearer token `12345`, send the header `Authorization: Bearer 12345`?